docker-compose.yml

```
@@ -1,11 +1,13 @@
 services:
 
   db:
-    image: postgres:12.0
+    image: postgres:16.0
     ports:
       - 5432:5432
     volumes:
       - ./database_dumps:/app/database_dumps
+    environment:
+      - POSTGRES_PASSWORD=postgres
```

To make `fab pull-staging-data` work:

```suggestion
    env_file:
          - .env
    environment:
      - POSTGRES_PASSWORD=postgres
```

Add `POSTGRES_HOST_AUTH_METHOD=trust` to `.env`, but not on the hosted environment